project.yml

```diff
@@ -1,86 +1,137 @@
+#########################################
+# THIS IS A TYPICAL project.yml TEMPLATE
+# most of the settings present here
+# are GO for mapping production data
+# but nonetheless user must consider
+# carefully every presented option
+#########################################
+
+
+# choose if you want to do FastQC:
 do_fastqc: False
+# choose if you want to do pairsamtools statistics:
 do_stats: True
     
 
+#######################################
+# provide paths to your raw input data (fastq-s):
+#######################################
 # Fastqs can be provided as:
 # -- a pairs of relative/absolute paths
 # -- sra:<SRA_NUMBER>, optionally followed by the indices of the first and
 # the last entry in the SRA in the form of "?start=<first>&end=<last>
-# [to implement] -- as a path to a folder with fastqs '<base_folder>', with the structure 
-# <base_folder>/<library_name>/<run_name>/, with each folder containing only
-# two fastq.gz files
 input:
     raw_reads_paths:
-        MATalpha_R1:
+        # substitute the location of your fastq files:
+        # an example of a 1-lane library:
+        your_library_name_1:
             lane1:
-                - ./test/fastq/MATalpha_R1/lane1/SRR2601842_1.fastq.gz
-                - ./test/fastq/MATalpha_R1/lane1/SRR2601842_2.fastq.gz
-#                 Alternatively, reads can be downloaded from SRA
-#                - sra:SRR2601842
-#                 - sra:SRR2601843?start=0&end=10000
-            lane2:
-#                 Files can also be provided as http/ftp/s3 links
-                - https://github.com/mirnylab/distiller-test-data/raw/master/fastq/MATalpha_R1/lane2/SRR2601843_1.fastq.gz
-                - https://github.com/mirnylab/distiller-test-data/raw/master/fastq/MATalpha_R1/lane2/SRR2601843_2.fastq.gz
-#                - ./test/fastq/MATalpha_R1/lane2/SRR2601843_1.fastq.gz
-#                - ./test/fastq/MATalpha_R1/lane2/SRR2601843_2.fastq.gz
-        MATalpha_R2:
+                - /path/to/your/fastq/from/library1/library1_R1.fastq.gz
```

Comment:
_R1/_R2 notation for the two sides of the library are a bit confusing (looks like replicate1/replicate2). However, that's the Illumina's notation, correct?

Reply:
You can really get a wide variety of naming schemes from different facilities, I think this doesn't matter...

Reply:
sides seems great! _side1/2